yarn.lock

```diff
@@ -9336,7 +9329,7 @@ cross-spawn@^7.0.0, cross-spawn@^7.0.2, cross-spawn@^7.0.3:
     shebang-command "^2.0.0"
     which "^2.0.1"
 
-crypto-js@4.2.0, crypto-js@^4.2.0:
+crypto-js@4.2.0, crypto-js@^3.3.0:
```

Comment:
Finally this is showing how we'd expect!

Reply:
is that right PR link?